src/data.rs

```diff
@@ -25,7 +25,7 @@ pub fn store_last_full_node_ws_in_db(db: Arc<DB>, last_full_node_ws: String) ->
 		LAST_FULL_NODE_WS_KEY.as_bytes(),
 		last_full_node_ws,
 	)
-	.context("Failed to write {key} data")
+	.context("Failed to write last full node ws to db")
```

Comment:
Why remove the key?  Its valuable info in determining the exact cause for error. Pls put it back.

Reply:
It was not applied anyway and there is no `key` variable in context

Reply:
Btw, I think there is no need to add explicit key here. We know what is that key because of this context.

Reply:
While it might be obvious from this pov, contextual info might not be enough in some cases in which we need to have specific values. 
example: going through logs of couple of hundred of LCs deployed at scale